Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -70,11 +70,11 @@ - (SentrySpanId *)startSpanWithName:(NSString *)name operation:(NSString *)opera
     return spanId;
 }
 
-- (void)measureSpanWithName:(NSString *)name
-                  operation:(NSString *)operation
-                    inBlock:(void (^)(void))block
+- (void)measureSpanWithDescription:(NSString *)description
+                         operation:(NSString *)operation
+                           inBlock:(void (^)(void))block
 {
-    SentrySpanId *spanId = [self startSpanWithName:name operation:operation];
+    SentrySpanId *spanId = [self startSpanWithName:description operation:operation];
```

Comment:
H: We will need a second method here with the original parameters in case we use this method to start a transaction.